keanu-project/src/main/java/io/improbable/keanu/network/NetworkWriter.java

```diff
@@ -0,0 +1,26 @@
+package io.improbable.keanu.network;
+
+import io.improbable.keanu.vertices.ConstantVertex;
+import io.improbable.keanu.vertices.Vertex;
+import io.improbable.keanu.vertices.bool.BoolVertex;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.nonprobabilistic.ConstantDoubleVertex;
+import io.improbable.keanu.vertices.intgr.IntegerVertex;
+
+import java.io.IOException;
```

Comment:
Might want to give these files a quick auto format in Intellij too.

Reply:
This doesn't seem to change if I autoformat in IntelliJ and Spotless doesn't touch it either?